server/athenian/api/slogging.py

```diff
@@ -0,0 +1,281 @@
+import argparse
```

Comment:
Could you briefly explain me the need for this as it seems a lot of work just for logging. I didn't dig depth into the code.

Reply:
Added the module docstring.